ddtrace/propagation/utils.py

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
no empty lines at the beginning of our modules. Task for the future: enforce a strict `flake8` ruleset